server/controllers/assess/assessmentsController.ts

```diff
@@ -61,22 +61,23 @@ export default class AssessmentsController {
 
   addNote() {
```

Comment:
I'm a bit confused about who should have the responsibility of adding a note now... we now have three `addNote` functions in three controllers - assessments, submittedApplications and Applications.

Reply:
I think the UI controllers should have maybe remained the same, because we have two ways in from the referrer and assessor side, and they are redirect back to different paths - but they should both talk to the assessment service

Reply:
Thanks!  This comment helped me realise that ultimately only the API endpoint needs changing.  The front end routes to adding notes aren't affected by this work and therefore there are minimal changes required to existing controllers, services and client methods.  Have rejigged and updated commit history as a result.